lxd/db/db_internal_test.go

```diff
@@ -242,7 +242,7 @@ func (s *dbTestSuite) Test_ImageExists_false() {
 
 func (s *dbTestSuite) Test_GetImageAlias_alias_exists() {
 	_ = s.db.Transaction(context.Background(), func(ctx context.Context, tx *ClusterTx) error {
-		_, alias, err := tx.GetImageAlias(ctx, "default", "somealias", true)
+		_, alias, err := tx.GetImageAlias(ctx, "default", "somealias", "container", true)
```

Comment:
To ensure we are getting both right, could you please introduce a similar test with a VM image alias?